koji_wrapper/base.py

```diff
@@ -53,6 +54,15 @@ def session(self, newsession):
         else:
             self.__session = koji.ClientSession(self.url)
 
+    @property
```

Comment:
I think this stuff should go in wrapper.py, that is the only place we use it

Reply:
re-factored to do just that.